apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -121,8 +121,32 @@ type BackendTLSPolicyValidation struct {
 	// not both. If CACertificateRefs is empty or unspecified, the configuration for
 	// WellKnownCACertificates MUST be honored instead if supported by the implementation.
 	//
-	// References to a resource in a different namespace are invalid for the
-	// moment, although we will revisit this in the future.
```

Comment:
Where did this go? Are references to a resource in a different namespace valid now?  If so, it should be explained how to make it happen, e.g. does it need  a referenceGrant?

Reply:
I moved it to here: https://github.com/kubernetes-sigs/gateway-api/pull/3994/files#diff-91c9d9844318bcde549158e45f9cf4adf9bef28a334f1796b95f36f741376bdfR143